src/style.css

```diff
@@ -1 +1,38 @@
-/* add styles here */
+/* #region head */
+html,
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Avoid using the `*` selector for setting properties like `margin`, `padding`, and `box-sizing`. It's more efficient to set these properties on specific elements that require them. The `*` selector has a performance cost because it applies the styles to every element in the DOM.